.gitignore

```diff
@@ -22,3 +22,4 @@ gardener
 
 // Artifacts from mutation testing
 *.cache
+.vscode
```

Comment:
I'd prefer you add any editor-specific directories to your global gitignore file